### PR TITLE
Misc scripts

### PR DESCRIPTION
**Pull request by jwasinger**
* remove `tee` which consumes a return code
* evm: invoke the right images.

### CODE REVIEW

File: benchmark_results_data/backup-2020-06-25-1593057958/evm_benchmarks.csv [hunk 1]
```diff
@@ -0,0 +1,108 @@
+engine, test_name, total_time, gas_used
```
Comment (hugo-dc) [inline benchmark_results_data/backup-2020-06-25-1593057958/evm_benchmarks.csv:1]: this is a backup csv file

File: benchmark_results_data/evm_benchmarks.csv [hunk 2]
```diff
@@ -1,108 +1,108 @@
 engine, test_name, total_time, gas_used
```
Comment (hugo-dc) [inline benchmark_results_data/evm_benchmarks.csv:1]: Also, since you are only changing script I think we shouldn't include benchmarks results?

Comment (jwasinger) [reply benchmark_results_data/evm_benchmarks.csv]: I've removed the unrelated files.

File: wasm-engines/bench_wasm_and_native.sh [hunk 3]
```diff
@@ -29,9 +29,7 @@ grep -E '^model name|^cpu MHz' /proc/cpuinfo > /benchmark_results_data/cpuinfo.t
 # fill rust code templates with input vectors. compile rust code to wasm and native. benchmark native
 # wasm will be further processed and benchmarked later
 cd /benchprep
-python3.8 benchnativerust_prepwasm.py --wasmoutdir="${WASM_FILE_DIR}" --csvresults="${CSV_NATIVE_RESULTS}" --rustcodedir="${RUST_CODE_DIR}" --inputvectorsdir="${INPUT_VECTORS_DIR}" |& tee prep-wasm-bench-native-run1.log
```
Comment (axic) [inline wasm-engines/bench_wasm_and_native.sh:32]: What was this `tee` thing doing?

Comment (jwasinger) [reply wasm-engines/bench_wasm_and_native.sh:32]: Idk. saving the build output to disk for later reference.

**Pull request merged**